scripts/world_bank/worldbank.py

```diff
@@ -0,0 +1,268 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" Fetches, cleans, outputs TMCFs and CSVs for all WorldBank development indicator codes provided in WorldBankIndicators.csv
+for all years for all countries provided in WorldBankCountries.csv """
+
+from absl import app
+import pandas as pd
+import numpy as np
+import itertools
+import requests, zipfile, io
+import re
+
+WORLDBANK_COL_REMAP = {
+  'Country Name':'CountryName',
+  'Country Code':'CountryCode',
+  'Indicator Name': 'IndicatorName',
+  'Indicator Code': 'IndicatorCode'
+}
+
+# The maximum number of constraints in the indicators to generate
```

Comment:
```suggestion
# The maximum number of constraints in the indicators to generate.
```

per http://google.github.io/styleguide/pyguide.html#386-punctuation-spelling-and-grammar